angular/core/components/thread_page/activity_card/discussion_edited_item_controller.coffee

```diff
@@ -1,6 +1,6 @@
-angular.module('loomioApp').controller 'DiscussionEditedItemController', ($scope) ->
-  discussion = $scope.event.discussion()
-  version = $scope.event.version()
+angular.module('loomioApp').controller 'DiscussionEditedItemController', ($scope, Records) ->
+  version    = Records.versions.find($scope.event.eventable.id)
+  discussion = Records.discussions.find(version.discussionId)
```

Comment:
Each event, in addition to serializing out the relevant records, now comes with an 'eventable' hash which looks like

```
eventable: {
  id: 1,
  type: 'discussion'
}
```

That makes it pretty easy to find an event's eventable through the `Records` interface.